custom/cowin/tests.py

```diff
@@ -139,7 +139,7 @@ def test_vaccination_payload(self, payload_doc_mock, connection_settings_mock):
                 "vaccine": "COVISHIELD",
                 "vaccine_batch": "123456",
                 "dose": 1,
-                "dose1_date": "2020-01-01",
+                "dose1_date": "01-01-2020",
```

Comment:
Guessing this is UK date format, not US date format? Why can't everyone just get onboard with ISO 8601?

Actually, just to make it more obvious, could you change the date to whatever 2020-12-31 is? (12-31-2020? 31-12-2020?)

Reply:
India format? DD-MM-YYYY :)

> Why can't everyone just get onboard with ISO 8601?

:(

Sure.

Reply:
Thanks for humouring me.

Yes! India format! South Africa format, Mexico format, France format ... nearly all of the countries ... nearly. :stuck_out_tongue_winking_eye: